providers/gitlab_test.go

```diff
@@ -39,129 +41,183 @@ func testGitLabBackend() *httptest.Server {
 			"groups": ["foo", "bar"]
 		}
 	`
+	projectInfo := `
+		{
+			"name": "MyProject",
+			"archived": false
+		}
+	`
 	authHeader := "Bearer gitlab_access_token"
 
 	return httptest.NewServer(http.HandlerFunc(
 		func(w http.ResponseWriter, r *http.Request) {
-			if r.URL.Path == "/oauth/userinfo" {
+			if r.URL.String() == "/oauth/userinfo" {
```

Comment:
Not sure I understand why this change was made? Can you remember?

I wonder if it's worth making this a switch statement rather than a series of `if` statements 🤔

Reply:
Not sure about why my team mate did this thing :shrug: